packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -127,7 +127,10 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     if (event is PointerMoveEvent) {
       final VelocityTracker tracker = _velocityTrackers[event.pointer];
       assert(tracker != null);
-      tracker.addPosition(event.timeStamp, event.position);
+      // PointerMoveEvent from platform's gesture up events are positionally redundant
+      // vs the last PointerMoveEvent. Do not use in velocity calculation.
+      if (event.sourceEventType != PointerChange.up)
+        tracker.addPosition(event.timeStamp, event.position);
```

Comment:
Three questions:
3) What does the OS send us if the user does 0,0 at t=0ms, 1,1 at t=16ms, stays still for another 16ms, and releases (still at 1,1) at t=32ms?